package.json

```diff
@@ -27,7 +27,9 @@
     "test:getExecutions": "jest tests/getExecutions.test.ts --verbose --runInBand",
     "test:getWallet": "jest tests/getWallet.test.ts --verbose --runInBand",
     "test:getWallets": "jest tests/getWallets.test.ts --verbose --runInBand",
-    "test:secret": "jest tests/secret.test.ts --verbose --runInBand"
+    "test:secret": "jest tests/secret.test.ts --verbose --runInBand",
+    "test:select": "npm run test -- --testNamePattern",
+    "build:example": "tsup examples/example.ts --format cjs,esm --watch examples/"
```

Comment:
@v9n, could you add instructions to README.md of using this example.js, e.g.
1. Should it be built before use?
2. command-line example including set env variables.

Reply:
okay, looks good but I added one more comment below.